src/App.js

```diff
@@ -1,24 +1,26 @@
-import logo from './logo.svg';
-import './App.css';
+import {
+  createBrowserRouter,
+  createRoutesFromElements,
+  Route,
+  RouterProvider,
+} from 'react-router-dom';
+import Home from './components/Home';
+import Categories from './components/Categories';
+import Books from './components/Books';
+
+const router = createBrowserRouter(
+  createRoutesFromElements(
+    <Route>
+      <Route index path="/" element={<Home />} />
+      <Route path="/books" element={<Books />} />
+      <Route path="/categories" element={<Categories />} />
+    </Route>,
```

Comment:
- [ ] [ OPTIONAL]
When I click on the categories link the whole navbar disappears. It will be a good practice if the navbar is made to stay at all pages. This will reflect more on the SPA feature.